index.js

```diff
@@ -199,9 +199,23 @@ const captureWebsite = async (url, options) => {
 			timeout: timeoutInSeconds
 		});
 		screenshotOptions.clip = await page.$eval(options.element, getBoundingClientRect);
+		if (options.inset) {
+			const {inset} = options;
```

Comment:
I would convert the shorthand here instead of doing it below:

```js
inset = typeof inset === 'number' ? {top: inset, right: inset, bottom: inset, left: inset};
```

Reply:
modified as your suggestion, thanks.